src/appwrite/category.tsx

```
@@ -0,0 +1,47 @@
+  }
+
+  async createCategory(ids: string[]) {
+    try {
+      return await this.databases.createDocument(
+        myConfig.REACT_APP_POSTS_DATABASE,
+        myConfig.REACT_APP_CATEGORIES_COLLECTION,
+        ID.unique(),
+        {posts: ids},
+      );
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)